src/main/java/underdogs/devbie/notice/service/NoticeService.java

```diff
@@ -0,0 +1,51 @@
+package underdogs.devbie.notice.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import underdogs.devbie.notice.domain.Notice;
+import underdogs.devbie.notice.domain.NoticeRepository;
+import underdogs.devbie.notice.dto.NoticeCreateRequest;
+import underdogs.devbie.notice.dto.NoticeDetailResponse;
+import underdogs.devbie.notice.dto.NoticeResponse;
+import underdogs.devbie.notice.dto.NoticeResponses;
+import underdogs.devbie.notice.dto.NoticeUpdateRequest;
+import underdogs.devbie.notice.expception.NoticeNotFoundException;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class NoticeService {
+
+    private final NoticeRepository noticeRepository;
+
+    @Transactional
+    public Long save(NoticeCreateRequest request) {
+        Notice savedNotice = noticeRepository.save(request.toEntity());
+        return savedNotice.getId();
+    }
+
+    @Transactional
+    public void update(Long id, NoticeUpdateRequest request) {
+        Notice notice = noticeRepository.findById(id).orElseThrow(NoticeNotFoundException::new);
+        notice.update(request.toEntity(id));
+    }
+
+    @Transactional
+    public void delete(Long id) {
+        noticeRepository.deleteById(id);
+    }
+
+    public NoticeResponses readAll() {
+        List<Notice> notices = noticeRepository.findAll();
+        return NoticeResponses.from(NoticeResponse.listFrom(notices));
+    }
+
+    public NoticeDetailResponse read(Long id) {
+        Notice notice = noticeRepository.findById(id).orElseThrow(NoticeNotFoundException::new);
```

Comment:
```suggestion
        Notice notice = noticeRepository.findById(id)
        .orElseThrow(NoticeNotFoundException::new);
```
.orElseThrow부분을 한줄로 이어쓸지 줄바꿈을 할 지 논의해서 통일하면 좋을 것 같습니다!